src/applications/appeals/996/tests/utils/helpers.unit.spec.js

```diff
@@ -51,7 +51,7 @@ describe('getEligibleContestableIssues', () => {
     attributes: {
       ratingIssueSubjectText: 'Issue 2',
       description: 'this is a deferred issue',
-      approxDecisionDate: parseDate(sub(date, { months: -1 })),
+      approxDecisionDate: parseDateWithOffset({ months: -1 }, date),
```

Comment:
This one stayed as `-1` because the original code was doing this, we missed this change when `parseDate(sub(...))` was added